src/module-2/components/searchbox/searchbox.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../css/global.css">
+    <link rel="stylesheet" href="../css/searchbox.css">
+</head>
+<body>
+    <section class="searchbox">
+        <div class="searchbox__container">
+            <div class="searchbox__top">
+                <div class="searchbox__results-found">7,618 results found</div>
+                <button type="submit" class="searchbox__favourite-button"></button>
+            </div>
+            <div class="searchbox__search">
+                <form action="#" class="searchbox__form"></form>
```

Comment:
you don't need a form here either. That's not a `form` with user input in general understanding